helpers/middleman/blog.rb

```diff
@@ -3,22 +3,50 @@
 
 module MiddlemanBlogHelpers
 
-  def blog_posts(order: nil)
+  def blog_posts(order: :latest_first)
     @blog_posts ||= {}
     return @blog_posts[order] if @blog_posts[order]
 
     @blog_posts[order] ||= case order
-      when :youngest_first
+      when :oldest_first
         data.blog.posts.values.sort do |a,b|
           a['created_at'] <=> b['created_at']
         end
-      when :oldest_first
+      when :latest_first
         data.blog.posts.values.sort do |a,b|
           b['created_at'] <=> a['created_at']
         end
       else
-        data.blog.posts
+        data.blog.posts.values
+      end
+  end
+
+  def blog_posts_for_menu
+    category_name_regexes = [
+      /Company News/,
+      /Investing Tips/,
+      /Release Notes/,
+      /Sharesight Features .* Tips/
+    ]
+
+    # get and cache a list of categories for the menu's list of blog posts
+    @__menu_blog_post_category_ids ||= blog_categories.select do |category|
```

Comment:
Why the `@__` syntax here?

Reply:
Ah, I just wanted to note it was local and not global, more private?

Maybe there's no precedent in Rails, but if one @portfolio never leaves the stack and one comes from the highest level controller, it's nicer for me to read and write if I can tell them apart.